moonsense/cli/download.py

```diff
@@ -0,0 +1,191 @@
+import os
```

Comment:
Shouldn't download.py be moved into the moonsense client so it can be called easily from within the python-sdk as well? Like this, you wouldn't be able to call download from the regular SDK.

Reply:
Good point -- I can expose some of the download logic in the main SDK as well.